{"cve_repaire_en": "", "cve_code_exp_url": "", "cve_id": "CVE-2019-9706", "cve_categories": ["category1", "category2"], "cve_code_detect_url": "", "cve_name": "cve_name", "cve_risk_level": "determine", "cve_detail_en": "Vixie Cron before the 3.0pl1-133 Debian package allows local users to cause a denial of service (use-after-free and daemon crash) because of a force_rescan_user error.", "cve_name_en": "cve_name_en", "edit_time": "Sat Oct 12 16:04:49 2019", "cve_code_des_url": "", "cve_other_info": "", "modify_time": "2019-03-21", "cve_detail": "", "references": [{"URL": "https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=809167", "Description": "MISC:https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=809167"}, {"URL": "https://packages.qa.debian.org/c/cron/news/20190311T170403Z.html", "Description": "MISC:https://packages.qa.debian.org/c/cron/news/20190311T170403Z.html"}, {"URL": "https://salsa.debian.org/debian/cron/commit/40791b93", "Description": "MISC:https://salsa.debian.org/debian/cron/commit/40791b93"}, {"URL": "https://lists.debian.org/debian-lts-announce/2019/03/msg00025.html", "Description": "MLIST:[debian-lts-announce] 20190321 [SECURITY] [DLA 1723-1] cron security update"}], "cve_repaire": "", "editor": "warcup", "add_time": "2019-03-11"}